src/domain/communication/room/Comments/CommentInputField.tsx

```diff
@@ -129,40 +176,35 @@ export const CommentInputField: FC<InputBaseComponentProps> = forwardRef<
       },
     });
 
-    const mentionableVCs = data?.lookup?.community?.virtualContributorsInRole?.filter(vc => {
-      return !isAlreadyMentioned(vc) && vc.profile.displayName.toLowerCase().includes(search.toLowerCase());
-    });
-
-    const mentionableUsers = data?.usersPaginated.users.filter(user => !isAlreadyMentioned(user));
```

Comment:
I suggest keeping only the vcInteractions in this PR, and if you think that the logic for the users should be revised, just create another thicket and we can look at it separately.